src/molecules/formfields/GoogleAutoCompleteField/index.js

```diff
@@ -51,6 +51,21 @@ class GoogleAutoCompleteField extends Component {
   updateAddress = () => {
     const place = this.state.autocomplete.getPlace();
     const addressData = this.formatAddressInfo(place.address_components);
+
+    if (!addressData.number) {
```

Comment:
This is admittedly a little brittle, since it assumes that the user is providing valid input. Basically, if the Places API fails to return a `street_number` field (for background info, see [this link](https://stackoverflow.com/a/48545776)), we attempt to parse it ourselves from the user's input. To do this, we try to parse a block of digits at the start of their input. If this fails, then we don't return a street number, and we're no worse off.